README.md

```diff
@@ -33,50 +30,63 @@ and try again.
 
 ### Manually
 
-#### How to build
+1. Download the latest [Hazelcast exporter JAR](https://github.com/zeebe-io/zeebe-hazelcast-exporter/releases)
 
-1. Build with Maven
-    ```
-	mvn clean install
-    ```
-
-2. Before you start the broker, copy the exporter JAR from the target folder into the lib folder of the broker.
+2. Copy the JAR into the broker folder `~/zeebe-broker-%{VERSION}/lib`
 
+3. Add the exporter to the broker configuration `~/zeebe-broker-%{VERSION}/config/zeebe.cfg.toml`.
     ```
-    cp exporter/target/zeebe-simple-monitor-exporter-%{VERSION}.jar ~/zeebe-broker-%{VERSION}/lib/
-	```
-
-	If you don't use the Hazelcast exporter yet then download the [Hazelcast exporter jar](https://github.com/zeebe-io/zeebe-hazelcast-exporter/releases) and copy it also into the lib folder.
-
-3. Register the exporters in the Zeebe configuration file `~/zeebe-broker-%{VERSION}/config/zeebe.cfg.toml`.
-    ```
-    [[exporters]]
-    id = "simple-monitor"
-    className = "io.zeebe.monitor.SimpleMonitorExporter"
-
     [[exporters]]
     id = "hazelcast"
     className = "io.zeebe.hazelcast.exporter.HazelcastExporter"
+    
+      [exporters.args]
+      # comma separated list of io.zeebe.protocol.record.ValueType
+      enabledValueTypes = "JOB,WORKFLOW_INSTANCE,DEPLOYMENT,INCIDENT,TIMER,VARIABLE,MESSAGE,MESSAGE_SUBSCRIPTION,MESSAGE_START_EVENT_SUBSCRIPTION"
```

Comment:
I'm not sure about that. I see that it's a lot to configure :sweat_smile: But in case what new types are added, you don't need to add them to the config :thinking:

Reply:
Would you prefer separate properties instead? Like the ES exporter do it.

For example:
``` 
      [exporters.args]
      exportJob = true
      exportWorkflowInstance=true
```

Reply:
> But in case what new types are added, you don't need to add them to the config 

Currently I have to,  because I need to explicit say which types

> Would you prefer separate properties instead? Like the ES exporter do it.

NO :sweat_smile: 

It should just export everything.
If I don't want to export variables then I can filter variables  :D